src/libs/intents/localMethods.ts

```diff
@@ -154,6 +156,7 @@ export const localMethods = (
     openAppOSSettings,
     isNativePassInstalledOnDevice,
     scanDocument,
-    isScannerAvailable: () => Promise.resolve(isScannerAvailable())
+    isScannerAvailable: () => Promise.resolve(isScannerAvailable()),
+    setTheme: setThemeIntent
```

Comment:
I'm still not sure what is the best way to handle the cozy-intent methods. But since we also use them internally, I don't think we should couple them like that, and keep them separate. And then localMethods is just something to glue the native and the web together